index.js

```diff
@@ -283,7 +283,8 @@ function inlineDirectiveRule(state, silent) {
     //
     // Markdown's design that paired `[]` doesn't need to be escaped
     // avoids "escape melaleuca" like `[\[\\[\\]\]]`. Clever design.
-    handler(state, content, dests, attrs, contentStart, contentEnd, state.pos, pos);
+    const isValidContent = handler(state, content, dests, attrs, contentStart, contentEnd, state.pos, pos);
+    if (!isValidContent) return false;
```

Comment:
Maybe we should check that the result of executing `handler()` is not false?

Because current change breaks backward compatibility